### ISSUE TITLE
[Request] Dead plugin sound

### ISSUE THREAD

**Issue by SladeHazard** [GC-spigot/Tokens#41]
Add more sound options to the plugin rn it seems very dead like when you purchase a enchant, there should be the experience sound and when a menu is opened there should be chest_open and once closed chest_close

**Comment by myiume** [comment]
So you want sounds when inventories close and open? o.o Adding such a feature will render a lot of issues with lower versions as compatibility, so an abstraction layer will be needed or handling the sounds per version. 

This will have to wait until we do a proper rework of the menu system.

**Comment by EwansXD** [comment]
An abstraction layer is not neccicary, just put a note above the section saying you will need to change the sound names if you are on 1.8 (1.9-1.16.4 sounds are same)

**Comment by Firosee** [comment]
Yeah more sounds would be perfect, and yeah just make people change the sounds for their server version.
